multiversx_sdk/network_providers/shared.py

```diff
@@ -5,3 +5,8 @@ def convert_tx_hash_to_string(tx_hash: Union[bytes, str]) -> str:
     if isinstance(tx_hash, bytes):
         return tx_hash.hex()
     return tx_hash
+
+
+def decimal_to_padded_hex(i: int) -> str:
```

Comment:
Can token compute be used instead, in the network provider?

Reply:
I added a new method to `TokenComputer`, let me know what you think.